package.json

```diff
@@ -21,5 +22,25 @@
   "bugs": {
     "url": "https://github.com/JorgeVQA/cypress-training/issues"
   },
-  "homepage": "https://github.com/JorgeVQA/cypress-training#readme"
+  "homepage": "https://github.com/JorgeVQA/cypress-training#readme",
+  "devDependencies": {
+    "@types/jest": "^27.4.1",
+    "typescript": "^4.6.3"
+  },
+    "compilerOptions": {
+        "target": "es5",
+        "skipLibCheck": true,
+        "strict": true,
+        "lib": ["es5", "dom"],
+        "types": [
+            "cypress",
+            "node"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    , "src"],
+    "exclude": [
+        "node_modules/"
+    ]
```

Comment:
The code between the lines 30 to 45, is already in the "tsconfig.json" file, those are not needed in this file.

Reply:
I will remove it, this is an exercise I was doing with Juan De La Hoz due the tsconfig file was not showing up as a .json.   Finally today worked so I will remove it here.   Thanks

Reply:
Fixed, I will send changes for you to check